### DISCUSSION TITLE
能不能插入参考文献呢 现在只能插入引注~

### DISCUSSION THREAD

**Discussion by guolizhang-tyler** [windingwind/zotero-better-notes#673] [Q&A]
![image](https://github.com/windingwind/zotero-better-notes/assets/116879961/6f895b35-74f1-4618-ae30-70a2271f95fa)

**Comment by 1584136307qq** [comment]
应该是可以的，可以看我新发的帖子

**Reply by huddao** [reply]
可否给个链接？
